docs-v2/components/homepage/GetStartedCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { Heading, Link, majorScale, Pane, Paragraph, useTheme } from "evergreen-ui"
+
+interface Props {
+  title: string,
```

Comment:
Seems like each of these cards pull the same Props interface? Might be worth exporting a common set of fields in a `types.ts` file, and then importing them in each of these cards. 

So you could do something like `interface Props extends BaseProps {}`